lib/netsuite/configuration.rb

```diff
@@ -407,5 +412,9 @@ def proxy(proxy = nil)
         attributes[:proxy]
       end
     end
+
+    def multi_tenant!
+      Thread.main[:netsuite_gem_multi_tenant] = true
```

Comment:
This is a global var so we can use `@multi_tenant` to store this value. Using a instance variable is better here: consistent with the rest of the config and leaks less configuration into global/thread-global state.